packages/react-ui-core/src/IncrementDecrementFilter/IncrementDecrementFilter.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class IncrementDecrementFilter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: this.props.count,
+    }
+    this.increment = this.increment.bind(this)
+    this.decrement = this.decrement.bind(this)
+  }
+  increment() {
+    const incrementValue = this.state.count + 1
+    this.setState({
+      count: incrementValue,
+    })
+  }
+  decrement() {
+    if (this.state.count > 1) {
+      const decrementValue = this.state.count - 1
+      this.setState({
+        count: decrementValue,
+      })
+    }
+  }
+  render() {
+    const { count } = this.state
+    const { theme, label, leftUnit, rightUnit, id } = this.props
+    return (
+      <div id={id} className={theme.container}>
+        <div className={theme.label}>
+          {label}
+        </div>
+        <div className={theme.componentContainer}>
+          <span
+            onClick={this.increment}
+            className={`leftUnit ${theme.contentInnards} ${theme.circle}`}
+          >
+            &#x002B;
+          </span>
+          <span
+            className={`${theme.contentInnards} ${theme.count}`}
+          >{`${leftUnit} ${count} ${rightUnit}`}</span>
+          <span
+            onClick={this.decrement}
+            className={`rightUnit ${theme.contentInnards} ${theme.circle}`}
+          >
+            &#x002D;
```

Comment:
regarding these html entities we should not be making assumptions about `+` or `-` signs although they are good defaults I would prefer it to be more generic and instead allow the user to inject their own button